Libraries/CommonFunctions.psm1

```diff
@@ -1869,6 +1869,14 @@ Function Publish-App([string]$appName, [string]$customIP, [string]$appGitURL, [s
         Write-LogErr "$appGitURL is not set"
         return $False
     }
+    # Install dependencies
+    $null = Copy-RemoteFiles -upload -uploadTo $customIP -port $VMSSHPort -files ".\Testscripts\Linux\utils.sh" `
+        -username $user -password $password 2>&1
+    $cmd = ". utils.sh && update_repos && install_package 'make build-essential'"
```

Comment:
is gcc-c++ specific to centos/redhat?

Otherwise, leave a single $cmd with all 3 packages, no hard to update it if it's already installed anyway.

Reply:
It is double checked that the package name is different between redhat and ubuntu.  To install the gcc-c++ on ubuntu, the Run-LinuxCmd will fail.

Reply:
you could have used ignore exit code flag? we don't check anyway for the exit code for deps install